### DISCUSSION TITLE
A75CCEA3-E1EC-2793-4AEC-508B4D9A7A8B

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#30] [General]
# A75CCEA3-E1EC-2793-4AEC-508B4D9A7A8B

This add-on adds a realistic, fully 3D model of the planetary nebula Abell 39. Abell 39 is one of the simplest planetary nebulae known, and deviates very little from a perfectly spherical shell.

https://celestia.mobi/addon?item=A75CCEA3-E1EC-2793-4AEC-508B4D9A7A8B&lang=en

**Comment by Dirgham2015** [comment]
O
